### PR TITLE
💄♻️🗑🎨 CMS Restyling (#547, #554, #640)

### PR DESCRIPTION
**Pull request by mistryrn**
Lots of stuff to digest here 🙈

### 💄 Updated the styling on the CMS pages to match latest designs (pre-"teal purge", colours will be updated in another PR).
* Updated CMS nav stying
* Changed background colour of CMS pages
* Updated a bunch of icons
* Updated Admin Edit Model form, Model Images, Model Variants
* Updated User Management Table
* Updated Notification banner styling
* Updated Model styles

### ♻️🗑🎨 Refactored a lot of the theme files, icons, and more
* ✨ Created a new Filter component to replace the search inputs used on different tables
* Refactored a lot of the icons
* Cleaned up some deprecated styles and components
* 🎨 Centralized all the button/pill styles back into `adminControlsStyles` – almost all of the buttons are now either a Pill or a HoverPill so updating buttons/button styles will be a breeze 🙌

### CODE REVIEW

File: ui/src/theme/adminControlsStyles.js [hunk 1]
```diff
@@ -4,79 +4,87 @@ import base from 'theme';
...
-  } else if (disabled) {
-    return { base: pillBlueDisabled, hover: pillBlueDisabled };
+    return { base: pillGrey, hover: frenchGrey };
   } else {
-    return { base: pillBlue, hover: '#58BAC9' };
+    return { base: pillBlueHighlight, hover: elm };
+  }
+};
+
+const pillBorderColour = ({ primary, warning, secondary, disabled, info }) => {
```
Comment (joneubank) [inline ui/src/theme/adminControlsStyles.js:60]: this works, could also write with a switch to stop the if/else ladder annoyance:
```
switch(true){
  case disabled:
    return pillDisabled;
  case primary:
    return pillBlue;
  default:
    return pillBlueHighlight;
}
```

Comment (joneubank) [reply ui/src/theme/adminControlsStyles.js:60]: (I know you didn't write it this way... I just get irked by long if/else chains lol)

File: ui/src/icons/PlusIcon.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { BaseSvg } from 'icons';
 
-export default ({ fill = '#fff', ...props }) =>
+export default ({ fill = '#FFFFFF', ...props }) =>
```
[the remark this answers is not in the document]
Comment (mistryrn) [reply ui/src/icons/PlusIcon.js:3]: I think this was just a copy/pase thing from replacing this file with the contents of another 😅

File: ui/src/theme/adminControlsStyles.js (hunk 1 above)
Comment (mistryrn) [reply ui/src/theme/adminControlsStyles.js:60]: Good call, I'll make this change in my colour updates PR

**Pull request merged**